README.md

```diff
@@ -8,9 +8,47 @@ This project provides an API for storing and retrieving learning resources that
 
 First, you'll need to configure a database. Once this project is deployed, we'll be using PostgreSQL. To set up a psql instance locally, follow the instructions for your OS:
 
-### Windows setup
+### Windows Setup
+
+1. [Install Chocolatey](https://chocolatey.org/docs/installation#installing-chocolatey) if you do not already have it installed.
+2. Add Chocolately and postgresql to your paths.   
+- Search --> Edit the [system environment variables](https://docs.microsoft.com/en-us/windows/desktop/shell/user-environment-variables) --> Environment Variables... --> Path --> Edit... --> New --> Enter `C:\ProgramData\chocolatey\bin` --> OK
+- New --> `C:\Program Files\PostgreSQL\10\bin` --> OK
+3. Set the System Variables.
+- Click the bottom New... -->
+- Connection Variable:
+    - Variable Name: `SQLALCHEMY_DATABASE_URI`
+    - Variable Value: `postgresql://aaron:password@127.0.0.1:5432/resources` (instead of aaron and password put what user name and password you want to use for postgresql) 
+- FLASK_APP Variable:
+    - Variable name: `FLASK_APP`
+    - Variable Value: `run.py`
+    - Optionally, enable debugging by setting the environment to development with the Variable Value: `development`
+4. Close out of Environment Variables and System Properites.
+5. In your powershell (admin), run `choco install postgresql10`. Follow the prompts.
+- powershell (admin) can be found by pressing `win + x`. 
+- Make sure you use the admin powershell throughout all the steps.
+6. Upgrade postgresql: `choco upgrade postgresql`.
+7. Start postgres: `psql -U postgres` you will see postgres-# in the terminal.
+8. Create your user with `CREATE USER name;` The terminal will print CREATE ROLE.
+9. Alter your role with permission to create a database with `ALTER USER name WITH CREATEDB CREATEROLE;` The terminal will print ALTER ROLE
+10. Check everything is in order with `\du`. The terminal will print a table with Role name, Atrributes, and Member columns. If your user is listed you are good to go.
+11. Quit with `\q` or `ctr + c`.
+12. Start psql with new username with `psql`. You should see `name=>`.
```

Comment:
We need a command that one can copy and paste. I assumed the command was `psql -U aaron` but that didn't work for me. There needs to be no ambiguity here.

Reply:
Also, I didn't even need to quit here. I was able to do `CREATE DATABASE resources OWNER aaron;` as the postgres user and that worked just fine. If we don't need the extra step of logging out of postgres and logging back in as the user, we should just take out steps 11 and 12 altogether.

Did you need to do this to get it to work?